Example/openxcframework/SendCanViewController.swift

```diff
@@ -62,15 +61,15 @@ class SendCanViewController: UIViewController, UITextFieldDelegate {
         
     
     }
-    func default_sendcan_rsp(_ rsp:NSDictionary) {
+    func defaultSendcanResponse(_ rsp:NSDictionary) {
```

Comment:
As above, The variable name should be defaultSendCanResponse

Reply:
Function Name done